static/admin/config.yml

```diff
@@ -37,10 +37,26 @@ collections: # A list of collections the CMS should be able to edit
         file: "content/copy/home.yml"
         description: "Copy for the home page"
         fields:
+          - name: homeCallout
+            label: "Left Callout"
+            widget: object
+            fields:
+
+                - name: title
+                  label: Home Callout Title
```

Comment:
Same with these, They will be subheader fields in the editor and should be obvious it is the title of the callout on the home page